cypress/e2e/wip/P2P/deleteAdFixedRate.cy.js

```diff
@@ -0,0 +1,76 @@
+import '@testing-library/cypress/add-commands'
+
+let fixedRate = 1.25
+let minOrder = 5
+let maxOrder = 10
+
+function verifyAdOnMyAdsScreen(fiatCurrency, localCurrency) {
+  cy.findByText('Active').should('be.visible')
+  cy.findByText(`Buy ${fiatCurrency}`).should('be.visible')
+  cy.findByText(`${fixedRate} ${localCurrency}`)
+  cy.findByText(
+    `${minOrder.toFixed(2)} - ${maxOrder.toFixed(2)} ${fiatCurrency}`
+  )
+}
+
+describe('QATEST-2482 - Delete Advert - Fixed Rate', () => {
+  beforeEach(() => {
+    cy.clearAllLocalStorage()
+    cy.clearAllSessionStorage()
+    cy.c_login({ user: 'p2pFixedRate' })
+    cy.c_visitResponsive('/appstore/traders-hub', 'small')
+  })
+
+  it('Should be able to delete newly created advert for fixed rate.', () => {
+    cy.c_navigateToDerivP2P()
+    cy.c_closeSafetyInstructions()
+    cy.findByText('Deriv P2P').should('exist')
+    cy.c_closeNotificationHeader()
+    cy.c_clickMyAdTab()
+    cy.c_checkForExistingAds().then((returnedValue) => {
```

Comment:
make name more meaningful
```suggestion
    cy.c_checkForExistingAds().then((adExists) => {
```

Reply:
This has been changed for better readability.